rootfs/usr/local/bin/build-kube-cfg

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eo pipefail
+
+KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
+KUBECONFIG_DIR="$(dirname "$KUBECONFIG")"
+EXTRA_ARGS=${@:1}
+GOMPLATE_EXTRA_ARGS=${EXTRA_ARGS:-$GOMPLATE_EXTRA_ARGS}
+
+mkdir -p ${KUBECONFIG_DIR}
+
+function write_kube_config() {
+  [[ "$KOPS_AWS_IAM_AUTHENTICATOR_ENABLED" == "true" ]] && template_iam_authenticator_cfg || kops_export_cfg
+}
+
+function kops_export_cfg() {
+  kops export kubecfg
+}
+
+function template_iam_authenticator_cfg() {
+  local KEYSET=$(mktemp)
+  aws s3 cp $KOPS_STATE_STORE/$KOPS_CLUSTER_NAME/pki/issued/ca/keyset.yaml $KEYSET
```

Comment:
`kops export kubecfg` generates a kubecfg with all the correct values by fetching data from its store. so it can fetch things like the CA, which it uses to populate kubecfg. Without first having this kubecfg, we don't have a straight forward way of getting the CA data needed to template our aws-iam-authenticator supporting kubecfg. https://github.com/kubernetes/kubernetes/issues/61572 covers not being able to access the CA data directly. This is the workaround I came up with that doesn't mean we already need a valid `kubecfg` with this info in it.

Reply:
Another option could maybe use https://github.com/kubernetes/kops/blob/master/docs/cli/kops_toolbox.md to template the config YAML file but we don't currently do that for the Kops manifest so I stuck with gomplate

Reply:
@osterman  Talking this over (🦆)  in Slack gave me an idea - see https://github.com/cloudposse/geodesic/pull/376/commits/e3f34ecd2292ef5c68ef07f7cc3562231cae2217

```
 ✓   (cpco-testing-admin) bin ⨠  rm -rf $KUBECONFIG
 ⧉  testing
 ✓   (cpco-testing-admin) bin ⨠  build-kube-cfg
kops has set your kubectl context to us-west-2.testing.cloudposse.co
Wrote Kube cfg to /dev/shm/kubecfg...
 ⧉  testing (⎈ |us-west-2.testing.cloudposse.co:default)
 ✓   (cpco-testing-admin) bin ⨠  cat $KUBECONFIG
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: <REDACTED>
    server: https://api.us-west-2.testing.cloudposse.co
  name: us-west-2.testing.cloudposse.co
contexts:
- context:
    cluster: us-west-2.testing.cloudposse.co
    user: us-west-2.testing.cloudposse.co
  name: us-west-2.testing.cloudposse.co
current-context: us-west-2.testing.cloudposse.co
kind: Config
preferences: {}
users:
- name: us-west-2.testing.cloudposse.co
  user:
    client-certificate-data: <REDEACTED>
    client-key-data: <REDACTED>
    exec:
      apiVersion: client.authentication.k8s.io/v1alpha1
      args:
      - token
      - -i
      - us-west-2.testing.cloudposse.co
      command: aws-iam-authenticator
      env: null
    password: <REDACTED>
    username: <REDACTED>
- name: us-west-2.testing.cloudposse.co-basic-auth
  user:
    password: <REDACTED>
    username: <REDACTED>
```

Note this config contains both the standard `kops export kubecfg` and also our `aws-iam-authenticator` command